proto/gz/msgs/navsat_with_covariance.proto

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2025 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package gz.msgs;
+option java_package = "com.gz.msgs";
+option java_outer_classname = "NavSatWithCovarianceProtos";
+
+/// \ingroup gz.msgs
+/// \interface NavSatWithCovariance
+/// \brief Data with covariance from a NavSat sensor
+/// This replaces the NavSat message.
```

Comment:
let's keep the old message for now. Instead of copying the contents of navsat.proto here, can you include it as a field in the new message? i.e.

```
import "gz/msgs/navast.proto";
...
message NavSatWithCovariance
{
  Header header                   = 1;  
  Navsat navsat                   = 2;  
  // Add new covariance fields below
  Float_V position_covariance     = 3;
  ...
}
```

Reply:
Won't the header get duplicated?

Reply:
ah ok that's not great. In that case, I'm ok with the current implementation as a replacement of the old msg.

Reply:
@caguero would you provide a recommendation?

Reply:
> @caguero would you provide a recommendation?

I'm in favor of copying the message as it is to avoid the header duplication.